src/setup/utils/metaMaskDownloader.ts

```diff
@@ -6,7 +6,13 @@ import * as path from "path";
 
 import StreamZip from "node-stream-zip";
 
-const defaultDirectory = path.resolve(__dirname, "..", "..", "..", ".metamask");
+export const defaultDirectory = path.resolve(
+  __dirname,
+  "..",
+  "..",
+  "..",
+  ".metamask"
+);
```

Comment:
Think I did mess up this long ago.
Is it not better to get rid of `__dirname`?
```suggestion
export const defaultDirectory = path.resolve("./.metamask");
```

Reply:
Yes, but its a breaking change now 😅

Reply:
using `.cache` sounds like a great idea, will keep files away from git, but will be pretty accessible

Reply:
using `.cache` sounds like a great idea, will keep files away from git, but will be pretty accessible

Reply:
True, and currently, users are storing metamask inside node_modules, inside dappeteer directory 😅

Reply:
But I didn't change anything so there is nothing to revert 😅 but issue about .cache sounds good to me^^